dsc_lib/src/discovery/mod.rs

```diff
@@ -6,9 +6,8 @@ mod discovery_trait;
 
 use crate::discovery::discovery_trait::ResourceDiscovery;
 use crate::{dscresources::dscresource::DscResource, dscerror::DscError};
-use regex::{RegexBuilder, Regex};
 use std::collections::BTreeMap;
-use tracing::{debug, error};
+use tracing::{error};
```

Comment:
Surprised clippy didn't complain about unnecessary braces

```suggestion
use tracing::error;
```